### DISCUSSION TITLE
Support for Symfony 7

### DISCUSSION THREAD

**Discussion by Atharaa** [1up-lab/OneupFlysystemBundle#293] [Q&A] [answered]
Hello do You have any idea when this bundle support symfony 7 ? Thanks

**Comment by bytehead** [comment, marked as answer]
It is already, see #291

**Comment by Atharaa** [comment]
Thanks for your answer but i see on composer.json requires symfony/config max ^6.0 it can't work with an update to sf7 ?

**Reply by bytehead** [reply]
Your're right. Only the dev dependencies were adjusted. Gonna fix that asap.

**Reply by Atharaa** [reply]
I do composer why-not symfony/config 7.0 et it's result to this response : 
oneup/flysystem-bundle 4.11.1  requires symfony/config (^4.4 || ^5.3 || ^6.0) so it doesn't work for sf7 update

**Reply by Atharaa** [reply]
Thanks

**Reply by bytehead** [reply]
go with 4.11.2 :)

**Reply by Atharaa** [reply]
Great thanks !
